lua/lspconfig/erlangls.lua

```diff
@@ -0,0 +1,31 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+
+configs.erlangls = {
+  default_config = {
+    cmd = { "erlang_ls" };
+    filetypes = {"erlang"};
+    root_dir = function(fname)
+      return util.root_pattern("rebar.config", "erlang.mk", ".git")(fname) or vim.loop.os_homedir()
+    end;
+    settings = {};
```

Comment:
Why empty settings?

Reply:
because of https://github.com/neovim/nvim-lspconfig/blob/80290ccc3a624b515541d1bc4e26257abf4951de/lua/lspconfig/sqls.lua#L11